cmd/backup-manager/app/backup/backup.go

```diff
@@ -123,6 +126,51 @@ func (bo *BackupOpts) backupDataToRemote(source, bucketURI string) error {
 	return nil
 }
 
+func (bo *BackupOpts) brBackupData(backup *v1alpha1.Backup) (string, string, error) {
+	args, path, err := constructBROptions(backup)
+	if err != nil {
+		return "", "", err
+	}
+	fullArgs := []string{
+		"backup",
+		fmt.Sprintf("%s", backup.Spec.Type),
+	}
+	fullArgs = append(fullArgs, args...)
+	output, err := exec.Command("br", fullArgs...).CombinedOutput()
```

Comment:
Did we consider using br as a library here? The main advantage is passing error codes in go memory without serialization.
An issue is handling logging. br may log inconsistently with the logging here. This is solvable by injecting the appropriate log formatter into BR.
The alternative of logging all the BR output here in a single message only after it completes has its own problems though.

Reply:
The code structure of br is not suitable to be used as a library, all the work is done in the `Run` function directly in the `cobra.Command`.

Reply:
I don't think it would be hard to make a library interface.

Reply:
Yes, but I prefer we make it work first, then later we can make a library interface for BR code and then we can use it in the operator, is it OK?

Reply:
Okay. Sometimes there are advantages to running a separate process.